### DISCUSSION TITLE
更新日志 - 22.03.5 - 20230728

### DISCUSSION THREAD

**Discussion by jjm2473** [istoreos/istoreos#913] [Announcements]
# 系统
1. OpenWRT 升级到 22.03.5。小版本升级，无需担心兼容性。其中需要注意的是：
   * curl 升级到 8.0.1

# 设备
* x86_64: 通过 Ventoy 启动的系统支持在线升级固件

**Comment by evoking6** [comment]
我先来

**Reply by maxguy66** [reply]
r5s在线更新并重启以后，红灯一直闪，上不去网，只能是拔了电再重启，重启后系统还是22.03.4，又可以正常上网了，试过两次都是这样。

**Reply by jjm2473** [reply]
再试一次

**Comment by weiwei52o** [comment]
x86在线升级完毕之后。硬盘休眠功能用不了啦。提示 （TypeError
Cannot read properties of null (reading 'trim')）

**Reply by jjm2473** [reply]
估计跟升级没什么关系，都没升级相关的组件。换浏览器或者清空浏览器缓存试试

**Reply by weiwei52o** [reply]
> 估计跟升级没什么关系，都没升级相关的组件。换浏览器或者清空浏览器缓存试试

谢谢

**Reply by tomato231** [reply]
我也出现了这个问题，一开始是能用的，后来我添加了一块硬盘就不行了....

**Reply by tomato231** [reply]
搞定了，原来要修改/etc/config/hd-idle

**Reply by jjm2473** [reply]
配置文件不兼容？那可以执行 `cp -a /rom/etc/config/hd-idle /etc/config/hd-idle`，然后应该就正常了，之后重新配置休眠

**Reply by ICE99125** [reply]
执行了 cp 还是不行呀？

**Reply by jjm2473** [reply]
@ICE99125 修复系统组件试试 https://github.com/istoreos/istoreos/wiki/FAQ 第2条

**Comment by xixiluhui520** [comment]
升级后正常，夸内核后升级再没有问题了。

**Reply by jjm2473** [reply]
小版本很难有什么问题

**Comment by shixin679** [comment]
22.03.4版本会频繁出现pppoe wan口重启；
这个版本也还存在这个问题，经常断网。
部分日志：
Sat Jul 29 10:59:04 2023 daemon.err nlbwmon[13083]: Error relocating /usr/sbin/nlbwmon: uloop_init: symbol not found
Sat Jul 29 10:59:04 2023 user.notice nlbwmon: Reloading nlbwmon due to ifup of wan (pppoe-wan)
Sat Jul 29 10:59:04 2023 user.notice wrtbwmon: Restart for ifup of wan (pppoe-wan)
Sat Jul 29 10:59:04 2023 user.notice firewall: Reloading firewall due to ifupdate of wan (pppoe-wan)
Sat Jul 29 10:59:04 2023 daemon.info netdata[12524]: RRDSET: chart name 'net_packets.pppoe_wan' on host 'iStoreOS' already exists.
Sat Jul 29 10:59:06 2023 user.notice wrtbwmon: Restart for ifupdate of wan (pppoe-wan)
Sat Jul 29 10:59:06 2023 daemon.info avahi-daemon[6595]: Leaving mDNS multicast group on interface pppoe-wan.IPv6 with address fe80::3eb3:8d1c:bb42:38e8.
Sat Jul 29 10:59:06 2023 daemon.info avahi-daemon[6595]: Joining mDNS multicast group on interface pppoe-wan.IPv6 with address 240e:398:11b:ba36:3eb3:8d1c:bb42:38e8.
Sat Jul 29 10:59:06 2023 daemon.info avahi-daemon[6595]: Registering new address record for 240e:398:11b:ba36:3eb3:8d1c:bb42:38e8 on pppoe-wan.*.
Sat Jul 29 10:59:06 2023 daemon.info avahi-daemon[6595]: Withdrawing address record for fe80::3eb3:8d1c:bb42:38e8 on pppoe-wan.
Sat Jul 29 10:59:06 2023 daemon.info avahi-daemon[6595]: Registering new address record for 240e:39a:13e:6bb0::1 on br-lan.*.
Sat Jul 29 10:59:06 2023 daemon.notice netifd: Interface 'wan_6' is now up
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: reading /tmp/resolv.conf.d/resolv.conf.auto
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using nameserver 223.5.5.5#53
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using nameserver 8.8.8.8#53
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using nameserver 2400:3200::1#53
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using nameserver 2001:4860:4860::8888#53
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for test
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for onion
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for localhost
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for local
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for invalid
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for bind
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: using only locally-known addresses for lan
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: read /etc/hosts - 12 names
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: read /tmp/hosts/dhcp.cfg01411c - 10 names
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq[1]: read /tmp/hosts/odhcpd - 4 names
Sat Jul 29 10:59:06 2023 daemon.info dnsmasq-dhcp[1]: read /etc/ethers - 0 addresses
Sat Jul 29 10:59:06 2023 user.notice firewall: Reloading firewall due to ifup of wan_6 (pppoe-wan)
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error loading shared library libubox.so.20210516: No such file or directory (needed by /usr/sbin/nlbwmon)
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_done: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_fd_add: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: avl_find: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: avl_delete: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: avl_init: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_timeout_cancel: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: __calloc_a: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: avl_insert: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_run_timeout: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_fd_delete: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: usock: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_timeout_set: symbol not found
Sat Jul 29 10:59:08 2023 daemon.err nlbwmon[14295]: Error relocating /usr/sbin/nlbwmon: uloop_init: symbol not found
Sat Jul 29 10:59:08 2023 user.notice nlbwmon: Reloading nlbwmon due to ifup of wan_6 (pppoe-wan)
Sat Jul 29 10:59:08 2023 user.notice wrtbwmon: Restart for ifup of wan_6 (pppoe-wan)
Sat Jul 29 11:00:00 2023 daemon.info netdata[12524]: RRDSET: chart name 'net_drops.pppoe_wan' on host 'iStoreOS' already exists.

设备在线截图：
![image](https://github.com/istoreos/istoreos/assets/124341659/9d992729-5eb6-4207-9a81-6988f4bde6d7)

重启后联网截图：
![image](https://github.com/istoreos/istoreos/assets/124341659/7d0f48a5-9b7b-464c-9d75-7bdcfb692fe2)

**Reply by Yuukipix** [reply]
是的，我也遇到了。我是PVE里装iStoreOS。
单独重启光猫，然后尝试拨号，无效。
重启iStoreOS，无效。
把光猫拔电，PVE宿主机拔电，再次启动后就能正常拨号。

**Reply by CKK803** [reply]
我也遇到同样问题了，R66S升级此版本后WAN口PPPOE频繁掉线重播，换成别的版本就没问题，希望下个版本尽快解决，我好刷回ISTORE OS的系统

**Reply by jjm2473** [reply]
都不知道什么原因，怎么修复，而且也没什么人反馈

**Reply by jjm2473** [reply]
我搭建了一个PPPOE服务器，然后用R66s拨号，iperf3压测了半个多小时，除了压测外还有正常流量的设备接入，都没有任何问题，也没报错日志，就是CPU温度飙到了75度

**Reply by maxguy66** [reply]
我的r5s更新后也没办法拨号，但有个临时解决办法，就是把wan口换成eth1

**Reply by jjm2473** [reply]
@maxguy66 换网线试试

**Reply by shixin679** [reply]
我更新了版本iStoreOS 22.03.5 2023080410，目前看7个小时都没有断网过了，后面再看看

![image](https://github.com/istoreos/istoreos/assets/124341659/798e96af-54c7-402f-9952-419900c6ea7a)
![image](https://github.com/istoreos/istoreos/assets/124341659/2c966663-c159-4f87-ac8b-9211f05310bd)

**Reply by yanjunhui** [reply]
r2s 也有同样的问题

**Reply by kbdancer** [reply]
x86同样有这个问题，wan一直重启，前几天用拨号正常，换成DHCP后就变这样了

**Comment by longyan2009** [comment]
(7月版本到最新0728)上传-都只能有1/6左右的上传速度 如：阿里云、百度、天翼云等。060914版本正常，尝试重置后，只设置宽带连接，情况仍然没改变~

**Reply by jjm2473** [reply]
是不是网卡协商到低速了？

**Reply by longyan2009** [reply]
不是 ，网络下载没任何问题 ，只上传有此问题 。正式又试了一次，重新刷机，只设置拨号上网，上网后，问题依旧，正式确定下载没问题，上传百度，天翼等速度只为正常的1/10，网络接口正常，无问题。0609或者之前版本没问题，一切正常，7月版本开始出现此问题。

**Reply by jjm2473** [reply]
OK，那就不知道什么原因了。

**Comment by a447948814** [comment]
建议恳求更新商店的软件版本，跟官方版本差太多了，理解作者比较忙，希望每个月可以抽一次时间更新下吧，或者自动从git检测更新下载？

**Reply by weiwei52o** [reply]
我也恳求更新，好多软件功能太老了。

**Reply by jjm2473** [reply]
可以自己pr的啊 https://github.com/linkease/istore-repo

**Reply by a447948814** [reply]
主要是istore的宗旨是 让小白也可以玩转opwrt哈哈哈，我不会PR，小白希望商店可以频繁更新软件 谢谢大佬！

**Comment by xixiluhui520** [comment]
这个版本更新后OpenClash只能解析到美国节点，不知道为什么，好多机场不能用了，老大这是怎么了

**Reply by jjm2473** [reply]
自己看看日志有没有报错，如果没有，那就问问openclash的作者。这次升级基本上只有curl可能影响到其他插件，只要curl命令没报错，那你遇到的问题就不可能是因为固件升级导致的。

**Reply by xixiluhui520** [reply]
对对对老大就是curl报错：【/tmp/openclash_last_version】下载失败：【curl: (35) ssl_handshake returned - mbedTLS: (-0x7280) SSL - The connection indicated an EOF】

**Reply by xixiluhui520** [reply]
root@iStoreOS:~# curl --version
curl 8.1.2 (x86_64-openwrt-linux-gnu) libcurl/8.1.2 mbedTLS/2.28.2 nghttp2/1.44.0
Release-Date: 2023-05-30
Protocols: file ftp ftps http https mqtt
Features: alt-svc HSTS HTTP2 HTTPS-proxy IPv6 Largefile SSL threadsafe

**Reply by xixiluhui520** [reply]
后来我干脆把curl搞到了最新，但是还是不行

**Reply by jjm2473** [reply]
这不是curl的错误，就是连不上openclash的下载服务器了。curl没问题

**Reply by xixiluhui520** [reply]
知道了！！那就是系统问题咯，谢谢老大

**Reply by jjm2473** [reply]
curl根本就不需要更新，是网络问题

**Reply by jjm2473** [reply]
毛线系统问题，中国的网络还不清楚咋回事吗

**Reply by xixiluhui520** [reply]
DNS问题

**Comment by juwrod** [comment]
拨号上网,端口转发不能用

**Reply by jjm2473** [reply]
试试istore里面的socat

**Reply by juwrod** [reply]
socat 也不行

**Reply by jjm2473** [reply]
估计运营商没开放端口，试试10000以上的端口，或者10000以内不常用的端口

**Reply by juwrod** [reply]
不行，我换其他路由测试一下

**Reply by jjm2473** [reply]
你确定自己的是公网ip？打开 https://www.ip138.com/ 看看ip是不是跟你拨号得到的ip一样
